libp2p/pubsub/pubsub.py

```diff
@@ -429,7 +435,12 @@ def handle_subscription(
         # Broadcast message
         for stream in self.peers.values():
             # Write message to stream
-            await stream.write(encode_varint_prefixed(raw_msg))
+            try:
+                await stream.write(encode_varint_prefixed(raw_msg))
+            except StreamClosed:
+                peer_id = stream.muxed_conn.peer_id
+                logger.debug("Fail to message peer %s: stream closed", peer_id)
+                self._handle_dead_peer(peer_id)
```

Comment:
Just a note: I'm thinking we can somehow extract the `try: ... except StreamClosed: ... self. _handle_dead_peer(peer_id)` to another function in the future.